src/main/java/com/garbagemule/MobArena/ScoreboardManager.java

```diff
@@ -29,6 +29,11 @@
         this.arena = arena;
         scoreboard = Bukkit.getScoreboardManager().getNewScoreboard();
         scoreboards = new HashMap<>();
+
+        if (arena.getSettings().getBoolean("show-health-under-name", true)) {
+            Objective health = this.scoreboard.registerNewObjective("health", "health", ChatColor.RED + "\u2764");
+            health.setDisplaySlot(DisplaySlot.BELOW_NAME);
```

Comment:
Does it make sense to move this down into the `initialize()` method as an additional method call like `resetHealth()`? If the objective isn't reset, won't the current values persist through to the next round? Or is that just something I made up in my mind working with kills? :P

Reply:
So, health is an odd one since it is auto-updated. When I was first testing it, everything seemed to work fine. But now, the exact problem you're describing is happening. I'll see what I can do :)